### PR TITLE
fixing string array bug

### PR DESCRIPTION
**Pull request by elodani**
ToArray() if you provide no arguments returns with Object[], now it returns
String[].

I also added template parameter (String) it is what we use it for anyways.

### CODE REVIEW

File: src/main/java/org/json/simple/ItemList.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 public class ItemList {
 	private String sp=",";
-	List items=new ArrayList();
+	List<String> items=new ArrayList<String>();
```
Comment (borsosdani) [inline src/main/java/org/json/simple/ItemList.java]: Please add comment

Comment (jakabanett) [inline src/main/java/org/json/simple/ItemList.java]: Please write comment!

File: src/main/java/org/json/simple/ItemList.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public List getItems(){
 	}
 	
 	public String[] getArray(){
-		return (String[])this.items.toArray();
+		return this.items.toArray(new String[0]);
```
Comment (jakabanett) [inline src/main/java/org/json/simple/ItemList.java]: Please write comment!

Comment (elodani) [reply src/main/java/org/json/simple/ItemList.java]: on it

**Pull request merged**